en/docs/guides/identity-federation/configure-ws-trust.md

```diff
@@ -46,141 +41,107 @@ To secure the Security Token Service:
-    .  
-    ![security-token-service-config]({{base_path}}/assets/img/tutorials/security-token-service-config.png)
-4.  Select **Yes** in the **Enable Security?** drop down and  select a
-    pre-configured security scenario according to your requirements. For
-    this tutorial, use **UsernameToken** under the **Basic Scenarios**
-    section.  
-    ![enable-security]({{base_path}}/assets/img/tutorials/enable-security.png)
+3. Expand **Inbound Authentication Configuration** > **Security Token Service Configuration** and click **Apply Security Policy**.
+    
+    ![apply security policy]({{base_path}}/assets/img/guides/ws-trust-apply-policy.png)
+
+4. Select `Yes` from the **Enable Security?** list and select a pre-configured security scenario according to your requirements.
+We will be using `UsernameToken` under the **Basic Scenarios** section for this tutorial.
 
     !!! note
-    
-        You can find further details about security policy scenarios from
-        the **view scenario** option **.**
-    
-        ![security-policy-scenarios]({{base_path}}/assets/img/tutorials/security-policy-scenarios.png)
-    
+        You can find further details about security policy scenarios from the **view scenario** option.
 
-5.  Click **Next**. The user domain and user group selection appears.
-	
-	!!! info	
-		Next steps may vary as per the security scenario that you have
-		chosen under point (5) above. Below is for **UsernameToken**
-		scenario.
+        ![enable-security]({{base_path}}/assets/img/guides/ws-trust-view-scenario.png)
 
-6.  Provide the required details as follows:
-    1.  Select **ALL-USER-STORE-DOMAINS**.
-    2.  Select the role you created to grant permission to access
-        secured service. In this example, the admin role is used.
-        Next, click **Finish**.
+5. Click **Next**.
 
-        !!! note
-                The **Select Domain** drop-down lists many domains. The listed
-                **User Groups** can vary depending on the domain selected.
-        
+    !!! info
+        The next steps may vary per the security scenario you have chosen under point (5) above. Below is for **UsernameToken** scenario.
 
-    	![select-domain]({{base_path}}/assets/img/tutorials/select-domain.png)
+6. Provide the required details as follows:
 
-7.  Click **Finish**.
-8.  Click **Ok** on the confirmation dialog window that appears.
-9.  Click **Update** to complete the process.
+    1. Select `ALL-USER-STORE-DOMAINS` as the domain.
+    2. Select the role you created to grant permission to access a secured service. In this example, the admin role is used.
+        ![select-domain]({{base_path}}/assets/img/guides/ws-trust-select-domain.png)
 
-Now STS is configured and secured with a username and password. Only
-users with the Admin role can consume the service.
+        !!! note
+                The **Select Domain** drop-down lists many domains. The listed **User Groups** can vary depending on the domain selected.
```

Comment:
`lists many domains` is very vague.. Can we improve on this? 

Is it something like `The Select Domain list shows the available domains.` 

About `The listed **User Groups** can vary depending on the domain selected.` - there are user groups in the Domain list?

Reply:
Updated with :arrow_down_small: 

The **Select Domain** drop-down lists the available domains. The listed **User Groups** can vary depending on the domain selected from the drop-down.